tests/Application/config/packages/twig_hooks.yaml

```diff
@@ -0,0 +1,29 @@
+sylius_twig_hooks:
+    hooks:
+        'sylius_admin.base#stylesheets':
+            wishlist_styles:
+                template: "@BitBagSyliusWishlistPlugin/Admin/_styles.html.twig"
```

Comment:
Templates names should be adjusted to the convention in Sylius in the next iteration